source/common/http/async_client_impl.h

```diff
@@ -44,6 +44,12 @@
 
 namespace Envoy {
 namespace Http {
+namespace {
+// Limit the size of buffer for data used for retries.
+// The default is set to 64KB. Since the existence of a buffer for
+// retries is non-trivial to the user, this value is not varied.
```

Comment:
This means that the user does not need to be aware of the existence of the litribuffer at this time because the buffer size is fixed length, but it is difficult to understand, so it will be removed.